README.md

```diff
@@ -55,6 +55,13 @@ To analyze JavaScript and TypeScript code, SonarLint requires Node.js executable
         "sonarlint.pathToNodeExecutable": "/home/julien/.nvm/versions/node/v11.12.0/bin/node"
     }
 
+### C and C++ analysis specific requirements
+
+To analyze C and C++ code, SonarLint requires compile commands json file:
+    {
+        "sonarlint.PathToCompileCommands": "/home/julien/repos/proj/compile_commands.json"
```

Comment:
Since the command is not yet implemented, I suggest to remove this documentation addition for now, and you can create a dedicated task ticket that we will do at then end.